src/Umbraco.Tests/Web/Mvc/RenderNoContentControllerTests.cs

```diff
@@ -39,10 +39,12 @@ public void Renders_View_When_No_Content_Published()
             mockUmbracoContext.Setup(x => x.Content.HasContent()).Returns(false);
             var mockIOHelper = new Mock<IIOHelper>();
             mockIOHelper.Setup(x => x.ResolveUrl(It.Is<string>(y => y == UmbracoPathSetting))).Returns(UmbracoPath);
-            var mockGlobalSettings = new Mock<IGlobalSettings>();
-            mockGlobalSettings.SetupGet(x => x.UmbracoPath).Returns(UmbracoPathSetting);
-            mockGlobalSettings.SetupGet(x => x.NoNodesViewPath).Returns(ViewPath);
-            var controller = new RenderNoContentController(new TestUmbracoContextAccessor(mockUmbracoContext.Object), mockIOHelper.Object, mockGlobalSettings.Object);
+            var globalSettings = new GlobalSettings()
```

Comment:
Isn't there a builder for this, or is this where you were thinking a builder was too much?

Reply:
Yep, I think the builder is too much, as this has a parameterless ctor and all default values make sense.